packages/gatsby-rehype-inline-images/src/gatsby-theme-try-ghost/styles/fluid.css

```diff
@@ -4,7 +4,6 @@
 /* ---------------------------------------------------------- */
 
 html.casper .post-content {
-    display: block;
```

Comment:
This line was introduced as a fix for #202. If you remove it, you will get zero-sized (regular) images again.